xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -57,11 +57,13 @@ unsigned int CRenderer::m_textureid = 1;
 CRenderer::CRenderer()
 {
   KODI::SUBTITLES::CSubtitlesSettings::GetInstance().RegisterObserver(this);
+  CServiceBroker::GetWinSystem()->GetGfxContext().RegisterObserver(this);
```

Comment:
This couples the overlay renderer with the graphic context/GUI. I am not sure this is something we would want

Reply:
there is currently no usable callbacks, the only one implemented is for GUI purpose
if there is a alternative way to get the callback welcome suggestions

Reply:
I wonder, isn't setVideoRect called when the resize happens? Couldn't this be used instead of going the observer route?

Reply:
i will check